statsd-client-async.opam

```diff
@@ -1,4 +1,4 @@
-opam-version: "1.2"
+opam-version: "1.3"
```

Comment:
Why did we need to change this?

Reply:
Ah, this is the version of Opam? I was thinking it was the version of _this_ package. Will roll that back, if so.

Reply:
Yeah this is the opam version. Now that I think about it, I don't think 1.3 exists (the latest two versions are 1.2 and 2.0)